src/arch/Sound/RageSoundDriver_WaveOut.h

```diff
@@ -9,6 +9,8 @@
 #include <windows.h>
 #include <mmsystem.h>
 
+constexpr int WAVEOUT_QTY_OF_BUFFERS=32;
```

Comment:
Let's make this a member variable (can be public so it can be referenced in the cpp). Also let's just call it `NUM_BUFFERS`

Reply:
👍